.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -1,6 +1,5 @@
 <component name="ProjectCodeStyleConfiguration">
   <state>
-    <option name="USE_PER_PROJECT_SETTINGS" value="true" />
```

Comment:
Ideally PR with the name `Change ui, improve ux` should not have updates like this